"preprocess" types not compatible with Svelte's types

The included types in this package are not compatible with the svelte compiler types. The main thing is the current `PreprocessOptions` type does not allow returned data from preprocessors.

It's easy enough to ignore the type error but since svelte exports the correct `preprocess` options type, there's no reason not to use it.

<details>
<summary>Sample error:</summary>

```sh
$ tsc --noEmit --allowJs --checkJs
rollup.config.js:80:14 - error TS2345: Argument of type '{ dev: boolean; emitCss: boolean; immutable: boolean; preprocess: { markup: ((options: { content: string; filename: string; }) => Processed | Promise<Processed>) | undefined; style: Preprocessor; }; preserveWhitespace: boolean; }' is not assignable to parameter of type 'Options'.
  Types of property 'preprocess' are incompatible.
    Type '{ markup: ((options: { content: string; filename: string; }) => Processed | Promise<Processed>) | undefined; style: Preprocessor; }' is not assignable to type 'PreprocessOptions'.
      Property 'markup' is incompatible with index signature.
        Type '((options: { content: string; filename: string; }) => Processed | Promise<Processed>) | undefined' is not assignable to type '(...args: any[]) => void'.
          Type 'undefined' is not assignable to type '(...args: any[]) => void'.

80       svelte(svelteOpts),
                ~~~~~~~~~~


Found 1 error.

error Command failed with exit code 1.
```
</details>

### Versions

- `rollup-plugin-svelte`: `5.2.3`
- `svelte`: `5.2.3`
- `typescript`: `3.9.7`

### References

- <https://github.com/sveltejs/svelte/blob/master/src/compiler/preprocess/index.ts#L7>
- <https://svelte.dev/docs#svelte_preprocess>

--- a/index.d.ts
+++ b/index.d.ts
@@ -1,6 +1,5 @@
 import { Plugin, RollupWarning } from 'rollup';
-
-interface PreprocessOptions extends Record<string, (...args: any[]) => void> {}
+import { PreprocessorGroup } from 'svelte/types/compiler/preprocess';
 
 interface Css {
   code: any;
@@ -55,7 +54,7 @@ interface Options {
    * Optionally, preprocess components with svelte.preprocess:
    * https://svelte.dev/docs#svelte_preprocess
    */
-  preprocess?: PreprocessOptions;
+  preprocess?: PreprocessorGroup | PreprocessorGroup[];
   // {
   //   style: ({ content }) => {
   //     return transformStyles(content);

